build-system/setup/__init__.py

```diff
@@ -438,6 +439,28 @@ def install_ideplugin (name):
 
 
 
+"""
+==============================================================================
+Name: install_vscode_support
+==============================================================================
+"""
+
+def install_vscode_support ():
+   print ('Adding vscode support...')
+
+   path_vscode_extensions = os.path.join (Path.home (), '.vscode', 'extensions')
+   path_src = os.path.join (PATH_BUILD_SYSTEM, 'setup', 'vscode')
+   path_dst = os.path.join (path_vscode_extensions, 'eurorack-blocks-vscode')
+
+   if os.path.exists (path_vscode_extensions) and os.path.isdir (path_vscode_extensions):
+      if os.path.exists (path_dst) and os.path.isdir (path_dst):
+         shutil.rmtree (path_dst)
+      shutil.copytree (path_src, path_dst)
+   else:
+      print (f"\033[33mwarning:\033[0m Can't find vscode extensions folder {path_vscode_extensions}.\nPlease make sure vscode is properly installed and run `erbb setup` again.")
```

Comment:
```suggestion
      print (f"\033[33mwarning:\033[0m Can't find vscode extensions folder {path_vscode_extensions}.\nPlease make sure vscode is properly installed and run `erbb setup --with-vscode-support` again.")
```

If we forgot to do that as well for Xcode, please don't hesitate to fix in that PR